pepper-apis/config/application.conf.ctmpl

```diff
@@ -18,6 +18,11 @@
     "port": {{$conf.Data.port}},
     "doLiquibase": {{$conf.Data.doLiquibase}},
     "threadTimeout": {{$conf.Data.threadTimeout}},
+    "auth0IpWhiteList": [
+    {{range $i, $ip := $conf.Data.auth0.ipWhiteList}}
+        {{if $i}},{{end}}"{{ $ip }}"
+    {{end}}
+    ],
     "auth0": {
```

Comment:
If someone has a better idea in how to generate an array from vault values in the template, please let me know!

Reply:
I tried it once for some elasticsearch config. I did the same thing but used `{{if gt $i 0}}`. What you have LGTM.